woocommerce/woocommerce-android
# [Better Tablet Products] Fix FAB position to be on the right/bottom of the screen

Closes: #10778


### Description

Fixes FAB on the 2-pane layout screen

### Testing instructions

* Make sure that with flag OFF, both tablet and phone work in single pane mode and FAB is located as used to be
* Make sure that with the flag ON, the tablet works in two-pane mode FAB is located at the bottom right

### Images/gif

<img width="611" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/05175f0c-51e5-4119-8930-3c393d8fcaf5">
